packages/query-core/src/types.ts

```diff
@@ -116,6 +117,7 @@ export type QueryFunctionContext<
   TPageParam = never,
 > = [TPageParam] extends [never]
   ? {
+      queryClient: QueryClient
```

Comment:
question: should we name this `client` instead of `queryClient` ?

I think `client` is the name we also have on the `Provider`. We usually show these examples

```
const queryClient = new QueryClient()

<QueryClientProvider client={queryClient}>...</QueryClientProvider>
```

so I’m thinking the new prop to destruct on the `QueryFunctionContext` should also be `client`, no?

```
useQuery({
  queryKey,
  queryFn: ({ client, signal }) => ...
})
```

Reply:
👍